tests/cases/fourslash/formatNoSpaceBetweenClosingParenAndTemplateString.ts

```diff
@@ -0,0 +1,10 @@
+/// <reference path="fourslash.ts"/>
+
+//// foo() `abc`;
+//// bar()`def`;
+
+format.document();
+verify.currentFileContentIs(
+`foo()\`abc\`;
+bar()\`def\`;`
```

Comment:
also add a test for a template with substitutions, since these are parsed diffrentlly e.g.

```ts
baz()`a${x}b`;
```